### DISCUSSION TITLE
Tunnel establishment failing due to CERT_REQ not sent as a part of IKE_AUTH request

### DISCUSSION THREAD

**Discussion by rahulmittalal** [strongswan/strongswan#2371] [Q&A] [answered]
I am facing issue with establishing a tunnel due to Authentication Failure. Upon analysing the trace, I observe that CERT_REQ is not send as a part of IKE_AUTH request from Initiator, leading to Auth Failure. 
Below is the Client side "swanctl.conf" output. To my knowledge, the configuration here looks fine. 
Could someone help me figure out what is wrong here, why isn't the CERT_REQ sent. 

======================
strongswan --version
Linux strongSwan U5.7.2/K4.18.0-372.32.1.el8_6.x86_64
======================

swanctl.conf:

connections {
        enrollment {
                local_addrs = 192.168.xx.xx
                remote_addrs = <fqdn>
                vips = 0.0.0.0,::
                local {
                        id = "/C=XX/O=xxxxxxx xxxxxxxx Inc./OU=Production/CN=xxxxxxxxxx"
                        auth = pubkey
                        certs = vendor_devcert.pem
                }
                remote {
                        id = %any
                        auth = pubkey
                }
                children {
                        enrollment-child {
                                local_ts = dynamic
                                remote_ts = 0.0.0.0/0,::/0
                                start_action = none
                                rekey_time = 0
                                esp_proposals = aes128-sha1
                                mode = tunnel
                                dpd_action = none
                        }
                }
                unique = no
                mobike = no
                version = 2
                dpd_delay = 30s
                dpd_timeout = 30s
                reauth_time = 0
                rekey_time = 24h
                proposals = aes256-sha256-modp1024
        }
}
secrets {
}

Please let me know if any additional info is required. 
P.S: I am unable to share the trace snippet due to restrictions.

**Comment by tobiasbrunner** [comment]
Do you actually have any CA certificates loaded?

**Reply by rahulmittalal** [reply]
yeah, the CA certs are loaded. 

[root@e9c84902523c ani]# swanctl --list-certs
 
List of X.509 End Entity Certificates
 
  subject:  "C=US, O=xxxxxxx xxxxxxx Inc., OU=Production, CN=xxxxxxx"
  issuer:   "C=US, O=xxxxxxx xxxxxxx Inc., CN=xxxxxxx xxxxxxx Inc. 2048-bit Root CA"
  validity:  not before Sep 19 00:00:00 2020, ok
             not after  Sep 19 23:59:59 2044, ok (expires in 7356 days)

**Reply by tobiasbrunner** [reply, marked as answer]
Clearly not a CA certificate (read the section title).
